pkg/engine/secrets/inspector.go

```diff
@@ -95,8 +95,10 @@ func NewInspector(
 	disableSecretsQuery bool,
 	executionTimeout int,
 	regexRulesContent string,
+	isCustomSecretsRegexes bool,
 ) (*Inspector, error) {
-	if disableSecretsQuery {
+	excludeSecretsQuery := isValueInArray("a88baa34-e2ad-44ea-ad6f-8cac87bc7c71", queryFilter.ExcludeQueries.ByIDs)
```

Comment:
I'd suggest getting the query ID from here https://github.com/Checkmarx/kics/blob/master/assets/assets.go#L12

Reply:
Great suggestion, @rogeriopeixotocx! 🚀